doc/source/technology_showcase_examples/techdemo-21/21-example-technology-showcase-buckling.py

```diff
@@ -0,0 +1,375 @@
+# used in the next step to generate the imperfections.
+# It is also an efficient way to check the completeness and
+# correctness of modeling.
+# To run the linear buckling analysis, a static solution with prestress effects
+# must be obtained, followed by the eigenvalue buckling solution using the
+# Block Lanczos method and mode expansion.
+
+# Define and solve linear buckling analysis
+mapdl.slashsolu()
+mapdl.outres("all", "all")
+mapdl.antype("BUCKLE")
+mapdl.bucopt("lanb", "10")
+mapdl.mxpand(10)
+output = mapdl.solve()
+print(output)
+
+# Plot total deformation of first and 10th mode
+mapdl.post1()
+mapdl.set(1, 1)
+mapdl.post_processing.plot_nodal_displacement("NORM", smooth_shading=True)
+mapdl.set(1, 10)
+mapdl.post_processing.plot_nodal_displacement("NORM", smooth_shading=True)
+#%%
+###############################################################################
+# Generate imperfections
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# If a structure is perfectly symmetric, nonsymmetric buckling does not occur
+# numerically, and a nonlinear buckling analysis fails because
+# nonsymmetric buckling responses cannot be triggered. In this problem,
+# the geometry, elements, and pressure are all axisymmetric.
+# It is not possible, therefore, to simulate nonaxisymmetric buckling with
+# the initial model. To overcome this problem, small geometric imperfections
+# (similar to those caused by manufacturing a real structure) must be
+# introduced to trigger the buckling responses.
+# Because the radius of the cylinder is 355.69 mm and the maximum
+# displacement of a mode shape is 1 mm, a factor of 0.1 is applied when
+# updating the geometry with mode shapes. The factor assumes the manufacturing
+# tolerance of the radius to be on the order of 0.1.
+mapdl.finish()
+mapdl.prep7()
+for i in range(1, 11):
+    mapdl.upgeom(0.1, 1, i, "buckling", "rst")  # Add imperfections as a tenth of each
+    # mode shape
+mapdl.finish()
+#%%
+###############################################################################
+# Run nonlinear static analysis on geometry with imperfections
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# The nonlinear buckling analysis is a static analysis performed after adding
+# imperfections with large deflection active (NLGEOM,ON), extended to a point
+# where the stiffened cylinder can reach its limit load.
+# To perform the analysis, the load must be allowed to increase using very
+# small time increments so that the expected critical buckling load can
+# be predicted accurately.
+# Note - as this is a buckling analysis, divergence is expected.
+
+mapdl.slashsolu()
+mapdl.antype("STATIC")
+mapdl.nlgeom("on")
+mapdl.pred("on")
+mapdl.time(1)
+mapdl.nsubst(100, 10000, 10)
+mapdl.rescontrol("define", "all", 1)
+mapdl.outres("all", "all")
+mapdl.ncnv(2)  # Do not terminate the program execution if the solution diverges
+mapdl.ignore_errors = True # in order for PyMAPDL to not raise an error 
```

Comment:
```suggestion
mapdl.ignore_errors = True  # in order for PyMAPDL to not raise an error 
```
Here is the correction